fbw-a32nx/src/systems/fmgc/src/guidance/GuidanceController.ts

```diff
@@ -112,6 +115,8 @@ export class GuidanceController {
 
     verticalProfileComputationParametersObserver: VerticalProfileComputationParametersObserver;
 
+    lastActiveLeg: Leg | null;
```

Comment:
What is this used for?

Reply:
This is used to update efis with the data based on the active leg computed by the LnavDriver.
If there's no more active leg, ETA, bearing and distance will be updated not to appear on the ND.

Reply:
Ah, my apologies, I missed that somehow 😄. Nevertheless, I think the property is redundant because we already store `activeGeometry` and `activeLegIdx` in the `GuidanceController`, so we can just derive the active leg there, rather than having it set in `LnavDriver`. This is just a minor thing though, so let me know if you have a different opinion.

Reply:
no worries, probably you're right on this one.
I've made the change, have a look.